app/views/admin/edition_images/_uploaded_images.html.erb

```diff
@@ -38,3 +38,28 @@
     margin_top: 0
   } %>
 </div>
+
+<% if edition.images %>
+  <ul class="govuk-list">
+    <% edition.images.each_with_index do |image, index| %>
+      <li class="app-view-edition-images__row govuk-grid-row">
```

Comment:
I notice you've given this `<li>` the class name `app-view-edition-images__row`, but it doesn't look like you've used it anywhere (e.g. there are no CSS or JS selectors for it).

Does this class name serve a purpose?